synapse/telepath.py

```diff
@@ -171,8 +188,28 @@ async def getAhaProxy(urlinfo):
     mesg = f'aha lookup failed: {host}'
     raise s_exc.NoSuchName(mesg=mesg)
 
+@contextlib.asynccontextmanager
+async def withTeleEnv():
+
+    async with loadTeleCell('/vertex/storage'):
+
+        yamlpath = s_common.getSynPath('telepath.yaml')
+        telefini = await loadTeleEnv(yamlpath)
+
+        certpath = s_common.getSynPath('certs')
+        s_certdir.addCertPath(certpath)
+
+        yield
```

Comment:
we need a `try: ... finally:` block here to ensure that we cleanup telefini and call delcertpath